hyperapp.d.ts

```diff
@@ -0,0 +1,149 @@
+declare module "hyperapp" {
+  export interface VNodeData {
+    [key: string]: any
+  }
+
+  export interface VNode {
+    tag: string
+    data: VNodeData
+    children: string | VNode | Array<VNode>
+  }
+
+  export type viewFunction<State, Actions> = (
```

Comment:
In general, it's better to declare interfaces than types. You can make an interface like,

```ts
interface View<S, A> {
     (state: S, actions: A): VNode;
}
```